apps/documents/assets/ChapterNavItem.jsx

```diff
@@ -14,7 +14,10 @@ function getErrorCount (props) {
 const ChapterNavItem = (props) => {
   return (
     <div className="commenting">
-      <button className="commenting__content commenting--toc__button button button--light button--small" type="button" onClick={props.onClick}>
+      <button
+        type="button"
+        className={`commenting__content commenting--toc__button button button--light button--small ${props.active ? 'active' : ''}`}
```

Comment:
You could use `classnames` here. I think we have it anyway: https://www.npmjs.com/package/classnames